cli/command/image/build/context.go

```diff
@@ -242,9 +242,9 @@ func getWithStatusError(url string) (resp *http.Response, err error) {
 	body, err := ioutil.ReadAll(resp.Body)
 	resp.Body.Close()
 	if err != nil {
-		return nil, errors.Wrapf(err, msg+": error reading body")
+		return nil, errors.Wrapf(err, fmt.Sprintf("%s: error reading body", msg))
 	}
-	return nil, errors.Errorf(msg+": %s", bytes.TrimSpace(body))
+	return nil, errors.Errorf("%s: %s", msg, bytes.TrimSpace(body))
```

Comment:
This could use `errors.New()` instead

```suggestion
	return nil, errors.New(msg+": %s", bytes.TrimSpace(body))
```

Reply:
Oh, never mind; ignore this one